### PR TITLE
Sensor scales

### PR DESCRIPTION
**Pull request by AlexLepauvre**
#### Reference issue

Implement https://github.com/mne-tools/mne-python/issues/12791

#### What does this implement/fix?
Add option to specify sensors scales in [_plot_sensors_3d](https://github.com/mne-tools/mne-python/blob/ac863e63e6233dd3ddb60e1058731f1839e55b02/mne/viz/_3d.py#L1456) and expose it as parameter in [mne.viz.plot_alignment](https://mne.tools/dev/generated/mne.viz.plot_alignment.html) and [mne.viz._brain._brain.py add_sensors](https://github.com/mne-tools/mne-python/blob/maint/1.8/mne/viz/_brain/_brain.py#L2702-L2808).
The implementation is the same as for sensor_colors

### CODE REVIEW

File: mne/utils/docs.py [hunk 1]
```diff
@@ -3997,6 +3997,19 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
...
+sensor_scales : int | float | array-like | dict | None
+    Scale to use for the sensor glyphs. Can be None (default) to use default scale.
+    A dict should provide the Scale (values) for each channel type (keys), e.g.::
+
+        dict(eeg=eeg_scales)
+
+    Where the value (``eeg_scales`` above) can be broadcast to an array of values with
+    length that matches the number of channels of that type. A few examples of this 
+    for the case above are the value ``10e-3``, a list of ``n_eeg`` values, or an NumPy ndarray of
+    shape ``(n_eeg)``.
```
Comment (larsoner) [inline mne/utils/docs.py:4010]:
```suggestion
    shape ``(n_eeg,)``.
```

File: mne/viz/_3d.py [hunk 2]
```diff
@@ -532,6 +532,7 @@ def plot_alignment(
     fig=None,
     interaction="terrain",
     sensor_colors=None,
+    sensor_scales=None,
```
Comment (larsoner) [inline mne/viz/_3d.py:535]: We've been trying to add these to functions as we add new params. Arguably it should go higher up but this is at least some progress
```suggestion
    *,
    sensor_scales=None,
```

File: mne/viz/_brain/_brain.py [hunk 3]
```diff
@@ -2736,6 +2737,9 @@ def add_sensors(
         %(max_dist_ieeg)s
         %(sensor_colors)s
 
+            .. versionadded:: 1.6
+        %(sensor_scales)s
+
             .. versionadded:: 1.6
```
Comment (larsoner) [inline mne/viz/_brain/_brain.py:2743]:
```suggestion
            .. versionadded:: 1.9
```

File: mne/viz/_brain/tests/test_brain.py [hunk 4]
```diff
@@ -549,6 +549,115 @@ def __init__(self):
...
+                [-30.08, 9.04, 28.23],
+                [-32.95, 9.04, 37.99],
+            ]
+        )
+        / 1000
+    )
+    proj_info.set_montage(
+        make_dig_montage(ch_pos=dict(zip(proj_info.ch_names, pos)), coord_frame="head")
+    )
+    if expectation is None:
```
Comment (larsoner) [inline mne/viz/_brain/tests/test_brain.py:643]: An easier pattern is:
```
expectation = expectation or nullcontext()
```
i.e., if `expectation` is None then use a `nullcontext`. Then you can do below no if/else just:
```
with expectation:
    brain.add_sensors(...)
```

**Review by larsoner — COMMENTED** [review]
If you look at the `style` CI you will see some problems the formatter couldn't auto-fix for you

For the changelog, can you add a `doc/changes/devel/12805.newfeature.rst` with something like
```
Added support for ``sensor_sizes`` to :meth:`mne.viz.Brain.plot_sensors` and :func:`mne.viz.plot_alignment`, by :newcontrib:`Alex Lepauvre`_.
```
and then add a URL for `Alex Lepauvre` to `doc/changes/names.inc`?

**Comment by AlexLepauvre** [comment]
[larsoner](https://github.com/larsoner) regarding my github URL, note that I also have another open PR in collaboration with a colleague. I have added it to both, I hope it won't cause any redundancies.

File: mne/viz/_brain/tests/test_brain.py (hunk 4 above)
Comment (larsoner) [reply mne/viz/_brain/tests/test_brain.py:646]: I think this is still pending

File: mne/viz/_brain/tests/test_brain.py [hunk 5]
```diff
@@ -549,6 +550,117 @@ def __init__(self):
...
+            sensor_scales=sensor_scales,
+        )
+    else:
+        with expectation:
+            brain.add_sensors(
+                proj_info,
+                trans=fname_trans,
+                sensor_colors=sensor_colors,
+                sensor_scales=sensor_scales,
+            )
```
Comment (larsoner) [inline mne/viz/_brain/tests/test_brain.py:660]:
```suggestion
    with expectation:
        brain.add_sensors(
            proj_info,
            trans=fname_trans,
            sensor_colors=sensor_colors,
            sensor_scales=sensor_scales,
        )
```

File: doc/changes/devel/12805.newfeature.rst [hunk 6]
```diff
@@ -0,0 +1 @@
+Added support for ``sensor_scales`` to :meth:`mne.viz.Brain.plot_sensors` and :func:`mne.viz.plot_alignment`, by :newcontrib:`Alex Lepauvre`_.
```
Comment (larsoner) [inline doc/changes/devel/12805.newfeature.rst:1]:
```suggestion
Added support for ``sensor_scales`` to :meth:`mne.viz.Brain.plot_sensors` and :func:`mne.viz.plot_alignment`, by :newcontrib:`Alex Lepauvre`.
```

**Review by larsoner — APPROVED** [review]
Marking for merge when green, thanks in advance @AlexLepauvre !

File: doc/changes/devel/12805.newfeature.rst [hunk 7]
```diff
@@ -0,0 +1 @@
+Added support for ``sensor_scales`` to :meth:`mne.viz.Brain.plot_sensors` and :func:`mne.viz.plot_alignment`, by :newcontrib:`Alex Lepauvre`.
```
Comment (larsoner) [inline doc/changes/devel/12805.newfeature.rst:1]:
```suggestion
Added support for ``sensor_scales`` to :meth:`mne.viz.Brain.add_sensors` and :func:`mne.viz.plot_alignment`, by :newcontrib:`Alex Lepauvre`.
```

File: mne/viz/_brain/tests/test_brain.py [hunk 8]
```diff
@@ -544,6 +545,109 @@ def __init__(self):
...
+            nullcontext(),
+        ),
+        (
+            {"seeg": ["k"] * 6},
+            {"seeg": [2] * 6},
+            nullcontext(),
+        ),
+    ],
+)
+def test_add_sensors_scales(sensor_colors, sensor_scales, expectation):
```
Comment (larsoner) [inline mne/viz/_brain/tests/test_brain.py:604]:
```suggestion
def test_add_sensors_scales(renderer_interactive_pyvistaqt, brain_gc, sensor_colors, sensor_scales, expectation):
```

**Pull request merged**